docs/source/install/installation.rst

```diff
@@ -294,144 +294,21 @@ If you open a new terminal for example, be sure to activate it first with::
 At this point, you can choose to read on for additional installation details and configuration options, or you can choose to start using
 AiiDA and go straight to the section :ref:`get started<get_started>`.
 
+.. _start_daemon:
```

Comment:
Before going here, should we let them check `verdi status` just as in the quick install section to make sure that everything is working. Or better, after starting the daemon, because otherwise that will show a cross

Reply:
thanks, added!